mods/ra/maps/allies-09a/allies09a.lua

```diff
@@ -0,0 +1,119 @@
+--[[
+   Copyright 2007-2020 The OpenRA Developers (see AUTHORS)
+   This file is part of OpenRA, which is free software. It is made
+   available to you under the terms of the GNU General Public License
+   as published by the Free Software Foundation, either version 3 of
+   the License, or (at your option) any later version. For more
+   information, see COPYING.
+]]
+
+lstReinforcements =
+{
+	actors = { "mcv"},
+	entryPath = { AlliedMCVEntry.Location, Unload1.Location },
+	exitPath = { AlliedMCVEntry.Location }
+}
+
+ExtractionHelicopterType = "tran"
+ExtractionPath = { HeliWP01.Location, HeliWP02.Location, HeliWP03.Location }
+
+kosyginType = "gnrl"
+contectKosygin = false
+
+MissionAccomplished = function()
+	Media.PlaySpeechNotification(Greece, "MissionAccomplished")
+end
+
+MissionFailed = function()
+	Media.PlaySpeechNotification(Greece, "MissionFailed")
+end
+
+InitialAlliedReinforcements = function()
+	Trigger.AfterDelay(DateTime.Seconds(1), function()
+		Media.PlaySpeechNotification(Greece, "ReinforcementsArrived")
+		Reinforcements.ReinforceWithTransport(Greece, "lst.reinforcement", lstReinforcements.actors, lstReinforcements.entryPath, lstReinforcements.exitPath)
+	end)
+end
+
+RescueFailed = function()
+	Media.PlaySpeechNotification(Greece, "ObjectiveNotMet")
+	Greece.MarkFailedObjective(KosyginSurviveObjective)
+end
+
+CreateKosygin = function()
+	Greece.MarkCompletedObjective(UseSpyObjective)
+	Media.PlaySpeechNotification(Greece, "ObjectiveMet")
+	kosygin = Actor.Create(kosyginType, true, { Location = KosyginSpawnPoint.Location, Owner = Greece })
+	Trigger.OnKilled(kosygin, RescueFailed)
+	ExtractObjective = Greece.AddPrimaryObjective("Extract Kosygin.")
+	Trigger.AfterDelay(DateTime.Seconds(1), function() Media.PlaySpeechNotification(Greece, "TargetFreed") end)
+end
+
+InfiltrateForwardCenter = function()
+	Trigger.OnInfiltrated(USSRFC, function()
+		if contectKosygin == false then
```

Comment:
`if not KosyginContacted then`

Reply:
Fixed at [c9bbea5](https://github.com/OpenRA/OpenRA/commit/c9bbea5b361b63990c5c073f6ac271692cc2ea62)